reference-stores/nuxt/queries/site.js

```diff
@@ -11,33 +11,37 @@ export const SITE_QUERY = `
         }
       }
     }
-    header: content(filter: { type: "componentHeader", handles: ["component-header"] }){
+    header: allContent(filter: { type: "componentHeader", handles: ["component-header"] }){
```

Comment:
For consistency & readability:

```suggestion
    header: allContent(filter: { type: "componentHeader", handles: ["component-header"] }) {
```